docs/UserGuide.md

```diff
@@ -173,7 +173,8 @@ The formats of the parameters used in the rest of the document are as follows:
 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijkl
 mnopqrstuvwxyz !"#$%&'()*+,-./:;<=>?@[\]^_`{|}~
   ```
-* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers up to 8 digits with 0 or 2 decimal places, with an optional `$` in front.
+* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers with up to 8 digits before the decimal point and with 0 or 2 decimal places.
+  An optional `$` may be included in front as well.
 * `DATE`, `DATE_FROM` and `DATE_TO` should be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```

Comment:
Inconsistent plurality between amounts and dates.
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be a non-negative number with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```
\- OR -
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should be non-negative numbers with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should be valid calendar dates in `dd/mm/yyyy` format, representing day, month, and year respectively.
```
However, the latter may suggest that multiple numbers or dates can be assigned to each amount/date.

Reply:
Another alternative using `each`:
```suggestion
* `AMOUNT`, `AMOUNT_FROM` and `AMOUNT_TO` should each be a non-negative number with up to 8 digits before the decimal point and with 0 or 2 decimal places.
  An optional `$` may be included in front as well.
* `DATE`, `DATE_FROM` and `DATE_TO` should each be a valid calendar date in `dd/mm/yyyy` format, representing day, month, and year respectively.
```